test/Mono.Linker.Tests.Cases/DataFlow/GenericParameterDataFlowWithoutMakeGeneric.cs

```diff
@@ -360,9 +345,8 @@ class TypeGenericRequirementsOnMembers<[DynamicallyAccessedMembers (DynamicallyA
 			public void PublicMethodsMethodParameter (TypeRequiresPublicMethods<TOuter> param) { }
 
 			public TypeRequiresPublicFields<TOuter> PublicFieldsMethodReturnValue () { return null; }
-			[ExpectedWarning ("IL2091", nameof (TypeRequiresPublicMethods<TOuter>))] // Return value
-			[ExpectedWarning ("IL2091", nameof (TypeRequiresPublicMethods<TOuter>))] // Compiler generated local variable
-			public TypeRequiresPublicMethods<TOuter> PublicMethodsMethodReturnValue () { return null; }
+			//[ExpectedWarning ("IL2091", nameof (TypeRequiresPublicMethods<TOuter>))]
+			//public TypeRequiresPublicMethods<TOuter> PublicMethodsMethodReturnValue () { return null; }
```

Comment:
We should either delete this or fix it (whichever makes sense).

Reply:
Fixed